app/views/research_sessions/questions/methodologies.html.erb

```diff
@@ -32,4 +32,41 @@
 
     <%= render partial: 'submit_step' %>
   <% end %>
+
+  <%= c('LivePreview') do %>
+    <%= c('HeadingOne', text: t("preview.able_to_consent.what_happens_in_this_research_session.heading")) %>
+
+    <%= c('Paragraph') do %>
+      <%= raw t("preview.able_to_consent.what_happens_in_this_research_session.paragraphs")[0] % [
+        c(
+          'Output',
+          text: @research_session.researcher_name,
+          field: 'researcher_name',
+          url: edit_link_for(:researcher_name)
+        ),
+        c(
+          'Output',
+          text: @research_session.researcher_job_title,
+          field: 'researcher_job_title',
+          url: edit_link_for(:researcher_job_title)
+        )
+      ] %>
+    <% end %>
+
+    <%= c('UnorderedList') do %>
+      <% t('preview.able_to_consent.what_happens_in_this_research_session.methodologies').each do |key, value| %>
+        <%= c('ChoicePreview', controlled_by: "[id=\"research_session[#{key}]\"]", isActive: (@research_session.methodologies.include?("#{key}") if @research_session.methodologies.present?)) do %>
```

Comment:
I'm not sure what this line is doing, can you walk me through it @joelcarr

Reply:
As talked offline, the isActive line is too long and should do a `&&` statement rather than a trailing `if`.